RecoHI/HiJetAlgos/plugins/HiFJGridEmptyAreaCalculator.h

```diff
@@ -0,0 +1,99 @@
+#ifndef HiJetBackground_HiFJGridEmptyAreaCalculator_h
+#define HiJetBackground_HiFJGridEmptyAreaCalculator_h
+
+
+// system include files
+#include <memory>
+#include <sstream>
+#include <string>
+#include <vector>
+
+// user include files
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "TMath.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+class HiFJGridEmptyAreaCalculator : public edm::EDProducer {
```

Comment:
Please change to inherit from a thread-safe EDProducer. More info here:

https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes#Stream_Module